src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -37,6 +41,63 @@ public AWSElasticBeanstalkHandler(IAWSClientFactory awsClientFactory, IOrchestra
             _optionSettingHandler = optionSettingHandler;
         }
 
+        /// <summary>
+        /// Elastic Beanstalk Windows deployments require a manifest file to be included with the application to be deployed.
+        /// This function creates the windows manifest file and adds it to the zip file that will be deployed to Beanstalk.
+        /// </summary>
+        public void SetupWindowsDeploymentManifest(Recommendation recommendation, string dotnetZipFilePath)
+        {
+            const string MANIFEST_FILENAME = "aws-windows-deployment-manifest.json";
+
+            var iisWebSiteOptionSetting = _optionSettingHandler.GetOptionSetting(recommendation, "IISWebSite");
+            var iisAppPathOptionSetting = _optionSettingHandler.GetOptionSetting(recommendation, "IISAppPath");
+
+            var iisWebSiteValue = _optionSettingHandler.GetOptionSettingValue<string>(recommendation, iisWebSiteOptionSetting);
+            var iisAppPathValue = _optionSettingHandler.GetOptionSettingValue<string>(recommendation, iisAppPathOptionSetting);
+
+            var iisWebSite = !string.IsNullOrEmpty(iisWebSiteValue) ? iisWebSiteValue : "Default Web Site";
+            var iisAppPath = !string.IsNullOrEmpty(iisAppPathValue) ? iisAppPathValue : "/";
+
+            var jsonStream = new MemoryStream();
+            using (var jsonWriter = new Utf8JsonWriter(jsonStream))
```

Comment:
Any concerns about the maintenance of this json writer (compared to using a POCO class with serialization)? I wouldn't know if this produces the correct structure.

Reply:
I have switched to using your suggestion